modules/sfp_darksearch.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:         sfp_darksearch
+# Purpose:      Searches the Darksearch.io Tor search engine for content related 
+#               to the domain in question.
+#
+# Author:      <bcoles[at]gmail[.]com>
+#
+# Created:     2019-05-11
+# Copyright:   (c) bcoles 2019
+# Licence:     GPL
+# -------------------------------------------------------------------------------
+
+import json
+import time
+import urllib
+from sflib import SpiderFoot, SpiderFootPlugin, SpiderFootEvent
+
+class sfp_darksearch(SpiderFootPlugin):
+    """Darksearch:Footprint,Investigate:Search Engines::Search the Darksearch.io Tor search engine for mentions of the target domain."""
+
+    # Default options
+    opts = {
+        'max_pages': 20
```

Comment:
Would be reeeeally nice if you could also implement the `fetchlinks` functionality :)

Reply:
Done